api/api-rule-violations-known.list

```diff
@@ -280,6 +280,7 @@ API rule violation: names_match,kubevirt.io/api/core/v1,KubeVirtConfiguration,SE
 API rule violation: names_match,kubevirt.io/api/core/v1,KubeVirtConfiguration,SMBIOSConfig
 API rule violation: names_match,kubevirt.io/api/core/v1,KubeVirtSpec,CertificateRotationStrategy
 API rule violation: names_match,kubevirt.io/api/core/v1,LunTarget,ReadOnly
+API rule violation: names_match,kubevirt.io/api/core/v1,LunTarget,Reservation
```

Comment:
I think you can easily avoid this violation, just make the field name and the json name match